README.md

```diff
@@ -24,10 +25,29 @@ with `rustup`:
 $ rustup component add llvm-tools-preview
 ```
 
-For BOLT, it's unfortunately more complicated. See [below](#bolt-installation) for BOLT installation
-guide.
+For BOLT, it is highly recommended to use [Docker](#docker).
+See [below](#bolt-installation) for BOLT installation guide.
+
+> BOLT and Docker support is currently *experimental*.
+
+## Docker
+
+To use latest `cargo-pgo` with Docker, you need to build the image first:
+
+```bash
+git clone https://github.com/Kobzol/cargo-pgo.git && cd cargo-pgo
+docker build -t cargo-pgo .
+```
+
+Then run this in your project directory to create a container:
+
+```bash
+docker run -v $(pwd):/workdir --rm -it cargo-pgo
```

Comment:
The resulting files will be owned by `root` by default :thinking: Maybe we should somehow mention setting a different user ID. But it's probably outside of scope for this README.